test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -1150,6 +1151,9 @@ private void RightClick(IInputElement element)
             });
 
             DispatcherUtil.DoEvents();
+
+            //Wait 3 seconds until the Click Right context menu is opened
+            Task.WaitAll(new Task[] { Task.Delay(3000) });
```

Comment:
Can we start with 1 or 2 seconds? I prefer a more conservative number since the test suite is time sensitive

Reply:
changed to 2 seconds in the next commit: https://github.com/DynamoDS/Dynamo/pull/13746/commits/6e8b4770a36bbf97ae46e7b29682b759899bcfc8